packages/libs/react-ui/src/components/Link/Link.css.ts

```diff
@@ -0,0 +1,32 @@
+import { sprinkles, vars } from '../../styles';
+
+import { style } from '@vanilla-extract/css';
+
+export const linkContainerClass = style([
+  sprinkles({
+    display: 'flex',
+  }),
+  {
+    selectors: {
+      '&:hover': {
+        textDecoration: 'none',
+      },
+      '&:active': {
+        color: vars.colors.$negativeContrast,
+      },
+      '&:visited': {
+        color: '#C870FF',
+      },
+    },
+  },
+]);
+
+export const iconContainerClass = style([
+  sprinkles({
+    display: 'flex',
+    marginX: '$2',
```

Comment:
I think it would be better to apply positioning in the parent using a gap so that there isn't margin on both sides of the icon